recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -24,9 +24,30 @@ ANDROID_VARIANT ?= "eng"
 SOC_FAMILY ?= "default"
 
 ANDROID_OUT_DIR_COMMON_BASE = ""
-DDK_KM_PREBUILT_MODULE ?= ""
-TARGET_PREBUILT_KERNEL ?= ""
-DDK_UM_PREBUILDS ?= ""
+
+python __anonymous () {
+    def create_param_str():
+         import os
+         path_not_exists_msg = "The %s does not exists."
+         d.setVar(pname,'')
+         value = d.getVar(vname);
+         if value != None:
+            if not os.path.exists(value):
+                  bb.fatal(path_not_exists_msg % value)
+            d.setVar(pname, "%s=%s" % (vname, value))
+
+    pname='PARAM_DDK_KM_PREBUILDS_MODULE'
+    vname='DDK_KM_PREBUILDS_MODULE'
+    create_param_str()
+
+    pname='PARAM_DDK_UM_PREBUILDS'
+    vname='DDK_UM_PREBUILDS'
+    create_param_str()
+
+    pname='PARAM_TARGET_PREBUILT_KERNEL'
+    vname='TARGET_PREBUILT_KERNEL'
+    create_param_str()
```

Comment:
Do you really need to copy-paste calls to that function? 

Why not to use list of variables?

Reply:
we can use the list of the variables, but , in this case, the list item has to be a structure (because every variable may be defined  and not defined), my opinion, it is more clear, to have 3 copy/paste blocks instead of array, because we have only 3 variables.